postgres-connector/resources/test/docker/vanilla-schema-image/Dockerfile

```diff
@@ -1,5 +0,0 @@
-FROM postgres:latest
```

Comment:
If we want to automate these tests in a pipeline we'll need to build on top of this idea and use something like kind. We should be able to use a regular postgres image (as is in the current chart) though will need to add the scripting for setup.

I suggest we include all this useful content you have working for now, then can discuss automation ideas after?

Reply:
That sounds like a plan there is still bits that need a bit of work.